src/com/engeto/CountryStats.java

```diff
@@ -0,0 +1,54 @@
+package com.engeto;
+
+import java.util.ArrayList;
+
+public class CountryStats {
```

Comment:
Navrh tejto triedy nie je najlepsie. 
Bud by som z toho udelal bezstavovu utility triedu s metodami List<Country> getHighestVat(List<Country> countries), podne pre lowest.

Alebo ked chceme drzat stav, tak by som plnenie listov highestVat a lowestVat riesil uz v konstruktoru, a getHighest() a getLowest() by boli bezparametrove.

Dovodom je mozna nekonzistencia objektu s takymto navrhom - obecne chceme tomu zabranit, aby nase objekty mohli byt v nekonzistentom stavu. Priklad:

```
CountryStats stats = new CountryStats();
stats.highestVat(countries);
// tu sme v nekonzistentom stave - co reprezentuje teraz nas objekt? je to taky polovicny
stats.lowestVat(countries2);
// poslal som tam iny zoznam, nas objekt berie highest a lowest z inych zdrojov
```

Dava ti to zmysel?

Reply:
Chapu, ze jsem udelal neco mezi. Vyplynulo to z toho, ze jsem potreboval udrzet informaci o highestVAT a lowestVAT. Predelal jsem to na bestavovou tridu.